README.md

```diff
@@ -391,7 +391,16 @@ For more detail about the behaviour of certain profiles, see the  [behaviour in
 
 ## Contributing
 
-Please read the [Contributing](https://github.com/finos/datahelix/blob/master/.github/CONTRIBUTING.md) guidelines before making any changes.
+1. Fork it (<https://github.com/yourname/yourproject/fork>)
+2. Create your feature branch (`git checkout -b feature/fooBar`)
+3. Read our [contribution guidelines](.github/CONTRIBUTING.md) and [Community Code of Conduct](https://www.finos.org/code-of-conduct)
+4. Commit your changes (`git commit -am 'Add some fooBar'`)
```

Comment:
Hi @aitana16 thanks for this.

could you update number 4 to be

```4. Commit your changes using our [Commit Guidelines](.github/CONTRIBUTING.md#commit-and-pr-messages) ```

your commit should probably look like 

'docs(#0): align README.md to FINOs project blueprint language'

because of our requirement to be using angular style commit messages. (we're considering removing this as it's an extra barrier to entry)

Reply:
hi @pdaulbyscottlogic, I just updated step 4 and committed it. Please let me know if there's anything else to be adjusted. Thanks